### ISSUE TITLE
V2.2.1 eight Unicode 9.0 emoji have the incorrect codepoints

### ISSUE THREAD

**Issue by eosrei** [Ranks/emojione#258]
Nice work on this update! Three of the new Unicode 9.0 emoji are missing compared to the [Emojipedia Unicode 9.0 list](http://emojipedia.org/unicode-9.0/). Probably not intentional.

1) 🥋 Martial Arts Uniform U+1F94B - http://emojipedia.org/martial-arts-uniform/
![Martial Arts Uniform](https://cloud.githubusercontent.com/assets/195061/15815469/21da2a9e-2b82-11e6-962b-aea1238fe7d2.png)

2) 🥊 Boxing Glove U+1F94A - http://emojipedia.org/boxing-glove/
![Boxing Glove](https://cloud.githubusercontent.com/assets/195061/15815482/3ee5edbc-2b82-11e6-8cc5-aff3b4f34e51.png)

3) 🥁 Drum With Drumsticks U+1F941 - http://emojipedia.org/drum-with-drumsticks/
![Drum With Drumsticks](https://cloud.githubusercontent.com/assets/195061/15815489/48ccde12-2b82-11e6-86d8-7d0cdf40335f.png)

**Comment by caseyahenson** [comment]
@eosrei Our "official" UC9 release within the next couple of weeks will address any final updates to UC9, including these. Thanks for your input!

**Comment by Naudiz** [comment]
These emojis are actually there, but they all have the wrong unicodes.

Boxing Glove has 1f93c (should be wrestlers)
Wrestlers has 1f93b (should be modern penthalon)
modern penthalon has 1f960 (should be fortune cookie in UNICODE 10.0)

drum with drumsticks has 1f93a (should be fencer)
fencer has 1f961 (this code shouldn't exist at all)

martial arts uniform has 1f93d (should be water polo)
water polo has 1f93e (should be handball)
handball has 1f93f (shouldn't exist at all)

This list might be incomplete, mayber there are even more filename-errors.

tl;dr: The new unicode 9.0 filenames are seriously messed up at the moment and need to get a bugfix asap.

**Comment by caseyahenson** [comment]
@Naudiz Thanks for your input! These artwork releases were preemptive which is why we didn't include them in the emoji.json data set. We'll have a new set of officially-named artwork as well as corresponding data coming out within the next couple of weeks, alongside our "official" UC9 release.

**Comment by eosrei** [comment]
FYI Emojipedia shows EmojiOne with Fencing in the Takeout Box location: http://emojipedia.org/takeout-box/
![Emojipedia Takeout Box](https://cloud.githubusercontent.com/assets/195061/15969735/252edb24-2ee6-11e6-9a11-15001e85d0a8.png)

**Comment by eosrei** [comment]
This seems to be the complete list of changes:
```
	renamed:    1f961.svg -> 1f93a.svg
	renamed:    1f960.svg -> 1f93b.svg
	renamed:    1f93b.svg -> 1f93c.svg
	renamed:    1f93e.svg -> 1f93d.svg
	renamed:    1f93f.svg -> 1f93e.svg
	renamed:    1f93a.svg -> 1f941.svg
	renamed:    1f93c.svg -> 1f94a.svg
	renamed:    1f93d.svg -> 1f94b.svg
```

I've rebuilt the [EmojiOne Color Font](https://github.com/eosrei/emojione-color-font) with the changes, but interestingly 🥋 Martial Arts Uniform U+1F94B,  🥊 Boxing Glove U+1F94A, and 🥁 Drum With Drumsticks U+1F941 still do not work as font characters. 😢 **FYI only, because nothing can done in this repo.** Linux and Firefox do not accept these values as valid codepoints, although Chrome with it's own font system does work. 
![Codepoints not working in Firefox](https://cloud.githubusercontent.com/assets/195061/15973613/4c442738-2ef9-11e6-8840-1f7368506035.png)

**Comment by caseyahenson** [comment]
Version 2.2.2 has been released to address these issues.

**Comment by jpchow26** [comment]
FYI the asset zip download from http://emojione.com/wp-content/uploads/assets/e1-png.zip still has these emojis named with the wrong codepoints
